Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1651,9 +1651,9 @@ private void ManageVisibility(bool isRestoringVisibility = false)
             {
                 if (clippedRenderer != null && !clippedRenderer.transform.IsChildOf(ScrollContainer.transform))
                 {
-                    if (!clippedRenderer.gameObject.activeSelf)
+                    if (!clippedRenderer.enabled)
```

Comment:
Tested the example scene out and found that the objects in this container did not render after a scroll.  The behavior also appears to be changed for the other items in the scrolling object collection examples.

| Scroll Before | Scroll Current|
|---|---|
| ![ScrollingObjBefore](https://user-images.githubusercontent.com/53493796/107076520-6614ce80-67a0-11eb-9a8a-434755067898.gif) |![ScrollingObjAfter](https://user-images.githubusercontent.com/53493796/107076494-5eedc080-67a0-11eb-9590-bba94eb66571.gif)

Reply:
Seems to be an issue with those prefabs being pre-initialized with those out of view game objects being disabled, so when the script is run currently, those objects never get enabled again 😅. Not super sure what the best way to address this while avoiding breaking the current experience for our users.

Reply:
Added new performance options that will let users toggle between the behaviors

![image](https://user-images.githubusercontent.com/39840334/107091937-91afad00-67d0-11eb-9d9c-a93e915a6117.png)